doc/source/conf.py

```diff
@@ -228,6 +228,15 @@
     'first_notebook_cell': '%matplotlib inline\n'
     'from pyvista import set_plot_theme\n'
     'set_plot_theme("document")\n',
+    'binder': {
+       'org': "pyvista",
+       'repo': "pyvista-tutorial",
+       'ref': "gh-pages",
+       'branch': "main",
```

Comment:
It seems a typo of sphinx-gallery(https://github.com/sphinx-gallery/sphinx-gallery/pull/970)
```suggestion
       'branch': "gh-pages",
```